sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-part-time.integration.service.ts

```diff
@@ -61,28 +61,28 @@ export class ECertPartTimeIntegrationService extends ECertIntegrationService {
     // Detail records
     // Calculated values
     const fileRecords = ecertRecords.map((ecertRecord) => {
-      const totalCSGPPTAmount = getDisbursementEffectiveAmountByValueCode(
+      const csgpPTAmount = getDisbursementEffectiveAmountByValueCode(
```

Comment:
I believe the total here make sense for the constants, as getDisbursementEffectiveAmountByValueCode will get the total awards amount with value code.

Reply:
As discussed, it makes more sense to remove the "total" from the name.